app/server/index.js

```diff
@@ -183,6 +183,16 @@ app.prepare().then(() => {
   };
   const keycloak = new Keycloak({store}, kcConfig);
 
+  // Nuke the siteminder session token on logout if we can
+  // this will be ignored by the user agent unless we're
+  // currently deployed to a subdomain of gov.bc.ca
+  //
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars-experimental
+  server.post('/logout', (req, res, next) => {
```

Comment:
renaming `req` to `_req` might turn off the eslint error?

Reply:
thanks, fixed